animation_nodes/nodes/sound/MIDI_Note_Info.py

```diff
@@ -0,0 +1,32 @@
+import bpy
+from ... base_types import AnimationNode, ListTypeSelectorSocket, VectorizedSocket
+
+class MidiNoteNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_MidiNoteInfoNode"
+    bl_label = "MIDI Note Info"
+    bl_width_default = 180
+
+    useNotesList: VectorizedSocket.newProperty()
+
+    def create(self):
+        self.newInput(VectorizedSocket("MIDINote", "useNotesList",
+            ("note", "note"), ("notes", "notes")))
+        self.newOutput(VectorizedSocket("Integer", "useNotesList",
+            ("Channel", "channel"), ("Channels", "channels")))
+        self.newOutput(VectorizedSocket("Integer", "useNotesList",
+            ("note", "note"), ("notes", "notes")))
+        self.newOutput(VectorizedSocket("Float", "useNotesList",
+            ("Time On", "timeon"), ("Times On", "timeson")))
+        self.newOutput(VectorizedSocket("Float", "useNotesList",
+            ("Time Off", "timeoff"), ("Times Off", "timesoff")))
+        self.newOutput(VectorizedSocket("Float", "useNotesList",
+            ("Velocity", "velocity"), ("Velocities", "velocities")))
+
+    def execute(self, note):
+        if self.useNotesList:
+            return [note.channel for note in note], [note.noteNumber for note in note], [note.timeOn for note in note], [note.timeOff for note in note], [note.velocity for note in note]
```

Comment:
You still have very long lines

Reply:
Could you tell me the limit ?

Reply:
Try to keep everything under 80-100 characters.